### PR TITLE
feat(theme): move aalborg theme into core

### PR DESCRIPTION
**Pull request by hypeJunction**
BREAKING CHANGE
Aalborg theme has been distributed among core views and plugins.
This may affect plugins that have relied on core styles,
or a certain priority of views

### CODE REVIEW

**Comment by hypeJunction** [comment]
Eh, how are these conflicts possible?

File: docs/guides/upgrading.rst [hunk 1]
```diff
@@ -237,6 +237,18 @@ required to make the old interface work.
 If your plugin is extending any of the views or relies on any actions in the notifications plugin,
 it has to be updated.
 
+Aalborg theme is now in core
+----------------------------
+
+Aalborg theme is no longer bundled with Elgg, instead all styles have been either moved to core views or to an appropriate core plugin.
+
+This change may affect themes and plugins that were developed without aalboarg stylesheets loaded or were relying on a certain priority of views with aalborg stylesheets loaded.
```
Comment (jdalsem) [inline docs/guides/upgrading.rst]: Aalborg typo

File: engine/lib/admin.php [hunk 2]
```diff
@@ -314,9 +314,10 @@ function _elgg_admin_init() {
...
 		elgg_register_menu_item('topbar', array(
 			'name' => 'administration',
+			'parent_name' => 'account',
 			'href' => 'admin',
-			'text' => elgg_view_icon('settings') . elgg_echo('admin'),
-			'priority' => 100,
+			'text' => elgg_echo('admin'),
+			'priority' => 101,
 			'section' => 'alt',
 		));
```
Comment (jdalsem) [inline engine/lib/admin.php]: i was wondering if it makes more sense to move this to the users_pagesetup? as it relies on the account parent...

File: engine/lib/users.php [hunk 3]
```diff
@@ -764,20 +764,32 @@ function users_pagesetup() {
...
+		));
+		
 		elgg_register_menu_item('topbar', array(
 			'name' => 'usersettings',
+			'parent_name' => 'account',
 			'href' => "settings/user/{$viewer->username}",
-			'text' => elgg_view_icon('settings') . elgg_echo('settings'),
-			'priority' => 500,
+			'text' => elgg_echo('settings'),
+			'priority' => 103,
```
Comment (jdalsem) [inline engine/lib/users.php]: can we have a bit more room in priorities? it is pretty hard to put a menu item between them in this way... just make them 150 and 200 or something

File: views/default/elements/buttons.css.php [hunk 4]
```diff
@@ -1,147 +0,0 @@
...
-	background: #eee;
-}
-
-/* Submit: This button should convey, "you're about to take some definitive action" */
-.elgg-button-submit {
-	color: white;
-	text-shadow: 1px 1px 0px black;
-	text-decoration: none;
-	border: 1px solid #4690d6;
-	background: #4690d6 url(<?= elgg_get_simplecache_url("button_graduation.png"); ?>) repeat-x left 10px;
```
Comment (jdalsem) [inline views/default/elements/buttons.css.php]: button_graduation image can also be removed

File: views/default/elements/buttons.css.php [hunk 5]
```diff
@@ -1,147 +0,0 @@
...
-.elgg-button-cancel:focus {
-	color: #444;
-	background-color: #999;
-	background-position: left 10px;
-	text-decoration: none;
-}
-
-/* Action: This button should convey a normal, inconsequential action, such as clicking a link */
-.elgg-button-action {
-	background: #ccc url(<?= elgg_get_simplecache_url("button_background.gif"); ?>) repeat-x 0 0;
```
Comment (jdalsem) [inline views/default/elements/buttons.css.php]: button_background as an image can be removed

File: views/default/elements/forms.css.php [hunk 6]
```diff
@@ -1,81 +1,79 @@
-<?php
+/* <style> /**/
```
Comment (jdalsem) [inline views/default/elements/forms.css.php]: can become a normal css file (if you change the image url)

File: views/default/page/elements/header.php [hunk 7]
```diff
@@ -8,7 +8,7 @@
 echo elgg_view('page/elements/header_logo', $vars);
 
 // drop-down login
-echo elgg_view('core/account/login_dropdown');
+//echo elgg_view('core/account/login_dropdown');
```
Comment (jdalsem) [inline views/default/page/elements/header.php]: why not remove this?

File: views/default/page/elements/header.php [hunk 8]
```diff
@@ -8,7 +8,7 @@
...
 
 // insert site-wide navigation
-echo elgg_view_menu('site');
\ No newline at end of file
+//echo elgg_view_menu('site');
```
Comment (jdalsem) [inline views/default/page/elements/header.php]: remove?

**Review by jdalsem — CHANGES_REQUESTED** [review]

**Comment by jdalsem** [comment]
Could you check if we can move some .php css files to normal css while at it?

i also was hoping that we could separate the media queries in separate views

**Comment by hypeJunction** [comment]
I prefer media queries closer to elements they overwrite.

**Comment by jdalsem** [comment]
the downside is that theming require you to copy all responsive behaviour...

the current way is that the media queries are at the bottom of the files... not super close to the elements they change... we could split it into 2 views...
buttons.css + buttons_responsive.css etc

**Comment by hypeJunction** [comment]
I hope we will throw most of them out eventually.

**Comment by jdalsem** [comment]
if you can resolve the conflicts and address my comments... i can give my thumbsup

**Comment by jdalsem** [comment]
don't want this lying around to long... (also conflicting with my walled garden PR), so this should go in before that PR

File: views/default/elements/buttons.css.php (hunk 4 above)
Comment (hypeJunction) [inline views/default/elements/buttons.css.php]: This is the old file. No longer there.

File: views/default/elements/forms.css.php (hunk 6 above)
Comment (hypeJunction) [inline views/default/elements/forms.css.php]: We will make it a normal css after loosing friendspicker.

File: views/default/elements/buttons.css.php (hunk 4 above)
Comment (jdalsem) [inline views/default/elements/buttons.css.php]: i meant the image file in core _graphics could also be removed

**Comment by hypeJunction** [comment]
I had to rebuild git index to resolve conflicts, but that picked up all the files that had bad chmod.

I don't want to rename all the .php files to .css just yet, because I want to break them down into smaller parts, and I think we will end up renaming them back to .php.

I changed the priorities of topbar items.

**Comment by hypeJunction** [comment]
Meh.

**Pull request closed without merging**